bundle/metadata/dependencies.yaml

```diff
@@ -7,6 +7,10 @@ dependencies:
   value:
     packageName: rook-ceph-operator
     version: ">=4.14.0 <4.15.0 || 4.15.0"
+- type: olm.package
+  value:
+    packageName: ocs-client-operator
+    version: ">=4.14.0 <4.15.0 || 4.15.0"
```

Comment:
@leelavg  Wouldn't this create issues for us when it comes to updating the client via the desired channel mechanism?

Reply:
Due to earlier work w/ operator condition odf-operator update itself will not get triggered if there will be unsupported clients.

So, native client-op subscription to be managed by odf-op is a safe operation.